MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -10,6 +10,118 @@ keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .ne
 
 # Pull To Refresh in MAUI DataGrid (SfDataGrid)
 
+To enable the `PullToRefresh` option in the data grid, set the [SfDataGrid.AllowPullToRefresh]() property to `true`. This allows the control to refresh the data source at runtime when the pull-to-refresh gesture is performed. 
+
+## Pull to refresh command
+
+The data grid can refresh its data during runtime by executing an `ICommand` bound to the [SfDataGrid.PullToRefreshCommand]() property. When a pull-to-refresh gesture is performed and the progress bar reaches 100%, this command is triggered to update the records in view.
+
+To indicate that a pull-to-refresh operation is in progress, set the [SfDataGrid.IsBusy]() property to `true` before starting the refresh. Once the refresh is complete, set this property back to `false`. You can also customize the duration of the pull-to-refresh animation by adding a delay in the code.
+
+To enable and perform the pull-to-refresh operation, refer to the following code example:
+
+{% tabs %}
+
+{% highlight Xaml %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       AllowPullToRefresh="True"
+                       PullToRefreshCommand="{Binding RefreshCommand}"
+                       ItemsSource="{Binding OrdersInfo}">
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+dataGrid.AllowPullToRefresh = true;
+
+Command RefreshCommand = new Command(ExecutePullToRefreshCommand);
+dataGrid.PullToRefreshCommand = RefreshCommand;
+ 
+private async void ExecutePullToRefreshCommand()
+{
+    this.dataGrid.IsBusy = true;
+    await Task.Delay(new TimeSpan(0, 0, 5));
+    viewModel.ItemsSourceRefresh();
+    this.dataGrid.IsBusy = false;
+}
+
+//ViewModel.cs
+public void ItemsSourceRefresh()
+{
+    int count = random.Next (1, 10);
+    for (int i = 1; i <= count; i++) 
+    {
+        this.OrdersInfo!.Insert(0, new OrderInfo()
+        {
+            OrderID = i,
+            CustomerID = this.customerID[this.random.Next(15)],
+            EmployeeID = this.random.Next(1700, 1800),
+        });
+    }        
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Pull To Refresh" src="Images\pull-to-refresh\net-maui-pull-to-refresh-slide-on-top.gif" width="404"/>
+
+### Transition mode
+
+To customize the progress view animation, adjust the [SfDataGrid.TransitionMode]() property. By default, the transition mode is set to SlideOnTop.
+
+* [SlideOnTop]()
+* [Push]()
```

Comment:
Remove the transition modes bullet marks

Reply:
removed